.github/workflows/create-release-tag.yaml

```diff
@@ -0,0 +1,19 @@
+name: Create release tag
+on:
+  push:
+    branches: [ develop ]
+
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.33.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        DEFAULT_BUMP: none
+        RELEASE_BRANCHES: develop
+        VERBOSE: false
```

Comment:
Does this read the latest version from the existing tags?
If not we may need INITIAL_VERSION too:
> INITIAL_VERSION (optional) - Set initial version before bump. Default 0.0.0.

Probably something like: `INITIAL_VERSION: 5.20.1`?

Reply:
Supposedly the bash script behind this GitHub action does check for existing tags. But it may be worth flipping `DRY_RUN` back on to make sure this does actually work as expected.

Reply:
I have now added a commit to keep in in `DRY_RUN` mode.